### DISCUSSION TITLE
Is there no version that supports kotlin 1.9.25?

### DISCUSSION THREAD

**Discussion by jamesrapadmi** [lupuuss/Mokkery#51] [Q&A] [answered]
I'm looking at the compatibility matrix [here](https://mokkery.dev/docs/Setup/#compatibility). And there's no entry for kotlin 1.9.25. Why is this?

**Comment by lupuuss** [comment]
Version 1.9.25 is not supported for a simple reason: it was released after Kotlin 2.0.0, by which time the Mokkery codebase had already migrated to K2. Releasing a version that supports Kotlin 1.9.25 would not have been very convenient. Additionally, the 1.9.25 release included only one change, so I didn’t think it was worth the effort. Is support for 1.9.25 necessary for you?

**Comment by jamesrapadmi** [comment]
We're currently on 1.9.25 as updating to kotlin 2 has been fraught with challenges. I was hoping to add Mokkery ahead of this migration to decouple these changes, but looks like that's not possible as no version works with 1.9.25.

**Comment by lupuuss** [comment]
Is downgrading to 1.9.24 an option for you?

**Comment by jamesrapadmi** [comment]
I guess maybe, TBH I'm focussing on getting everything ready to update to 2.0. Although I believe 1.9.25 has much better interoperability with Android Studio than 1.9.24

**Comment by lupuuss** [comment]
I understand. Is there anything I could do for you?

**Comment by jamesrapadmi** [comment]
Can you release a version that is compatible with 1.9.25?
Otherwise, we'll have to integrate alongside the main kotlin 2 migration.

**Comment by lupuuss** [comment]
I could release version `1.7.0` with Kotlin bumped to `1.9.25`, so it won't contain any bug fixes that occurred in `2.*` versions. Is this ok for you?

**Comment by jamesrapadmi** [comment]
Yeah that'd be really helpful thanks

**Comment by lupuuss** [comment, marked as answer]
I've published the `1.9.25-1.7.0` version. Also, I've updated the compatibility matrix.

**Comment by jamesrapadmi** [comment]
Thank you
